### DISCUSSION TITLE
Replace Bromite with Chromite?

### DISCUSSION THREAD

**Discussion by technoasatruar** [privacytoolsIO/privacy-tools#11] [Submit Privacy Tools]
Bromite hasn't been updated in nearly a year and one of the devs forked it, it seems odd to keep this outdated browser given how dangerous that could be for security.

**Comment by DJCrashdummy** [comment]
IMHO it is also worth mentioning [Mull](https://f-droid.org/packages/us.spotco.fennec_dos/) as you can see it very loosely as the mobile pendant to LibreWolf for the desktop.
